### DISCUSSION TITLE
Full-stack project

### DISCUSSION THREAD

**Discussion by Frikadellios** [nuejs/nue#82] [Ideas]
Have a question about something like full stack project with Nue.js/Rsspress/Bun.sh/Drizzle/Hono or something like that...

Localization ready shef 🤣

**Comment by tipiirai** [comment]
Nue.js currently under intense development (in private repository as for now) and a minimum viable product should be out next month. It's a strong alternative to Next.js and Astro. Full stack projects are totally possible then.

**Comment by tipiirai** [comment]
Please check out the [Announcement of Nuekit](https://github.com/nuejs/nue/discussions/108)
